server/util/approxlru/approxlru.go

```diff
@@ -251,7 +251,7 @@ func (l *LRU[T]) evictSingleKey() (*Sample[T], error) {
 		oldGlobalSizeBytes := l.globalSizeBytes
 		l.mu.Unlock()
 
-		log.Debugf("Evictor attempting to evict %q (last accessed %s)", sample.Key, l.clock.Since(sample.Timestamp))
+		//log.Debugf("Evictor attempting to evict %q (last accessed %s)", sample.Key, l.clock.Since(sample.Timestamp))
```

Comment:
You can just delete this?

Reply:
I was thinking this might be needed when debugging tests?